### PR TITLE
Add logging for middleware server

### PR DESCRIPTION
**Pull request by dpakach**
Fixes https://github.com/owncloud/owncloud-test-middleware/issues/43
Add logging for easy debugging

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
very needed feature. 🎩 
LGTM 👍🏼

File: src/app.js [hunk 1]
```diff
@@ -58,13 +60,13 @@ app.use('/execute', async (req, res) => {
   const reqStep = new Step(token, pattern, table)
   const stepDef = testContext.getMatch(reqStep)
 
-  console.log('Executing new step:\n', reqStep)
+  log.info('Executing new step: ' + JSON.stringify(reqStep))
```
Comment (kiranparajuli589) [inline src/app.js:63]:
```suggestion
  log.info('Executing new step:\n ' + JSON.stringify(reqStep))
```
newline would be nice here

**Comment by kiranparajuli589** [comment]
very needed feature. 🎩 
LGTM 👍🏼

**Review by individual-it — APPROVED** [review]

File: src/app.js (hunk 1 above)
Comment (dpakach) [reply src/app.js:63]: not a good idea IMO to have line breaks in logs.

**Pull request merged**